### PR TITLE
Handled 202 Response Code for ARM Restart API

### PR DESCRIPTION
**Pull request by AmrutaKawade**
Handled 202 Response Code for ARM Restart API for container apps

### CODE REVIEW

**Review by aksm-ms — APPROVED** [review]

**Comment by aksm-ms** [comment]
@asranja Looks good to me, please review once.

File: packages/appservice-rest/src/Arm/azure-app-service.ts [hunk 1]
```diff
@@ -65,14 +65,14 @@ export class AzureAppService {
 
             console.log("Restarting app service: " + this._getFormattedName());
             var response = await this._client.beginRequest(webRequest);
-            console.log(`Restart response: ${JSON.stringify(response)}`);
+            console.debug(`Restart response: ${JSON.stringify(response)}`);
```
Comment (aksm-ms) [inline packages/appservice-rest/src/Arm/azure-app-service.ts:68]: use core.debug in all debug logs.

File: packages/appservice-rest/src/Arm/azure-app-service.ts [hunk 2]
```diff
@@ -65,7 +65,21 @@ export class AzureAppService {
 
             console.log("Restarting app service: " + this._getFormattedName());
             var response = await this._client.beginRequest(webRequest);
-            if(response.statusCode != 200) {
+            console.log(`Restart response: ${JSON.stringify(response)}`);
+            if (response.statusCode == 200) {
+                console.log('Deployment passed');
```
[the remark this answers is not in the document]
Comment (AmrutaKawade) [reply packages/appservice-rest/src/Arm/azure-app-service.ts:70]: done

File: packages/appservice-rest/src/Arm/azure-app-service.ts (hunk 1 above)
Comment (AmrutaKawade) [reply packages/appservice-rest/src/Arm/azure-app-service.ts]: done

**Review by asranja — APPROVED** [review]

**Pull request merged**